### ISSUE TITLE
Pages do not scroll normally when scrolling after turning pages using the indicator.

### ISSUE THREAD

**Issue by elixirevo** [Hejtmus/svelte-fullpage#54]
**Describe the bug**
Pages do not scroll normally when scrolling after turning pages using the indicator.

**To Reproduce**
1. Go to '[example](https://hejtmus.github.io/svelte-fullpage/)'
2. Click the third dot on the right indicator
4. Down the page with a mouse scroller.
5. There are no more pages to go down based on the third page, but the second page is scrolled down based on the first page.

**Desktop (please complete the following information):**
- Device: MacBook
 - OS: MacOS 13.2.1
 - Browser chrome
 - Version  112.0.5615.49

**Additional context**
In the FullpageActivity function, the toPage function assigns a pageId value to the activePage variable to resolve the problem.

**Comment by Hejtmus** [comment]
Hey @elixirevo,

thank you for great issue and pull request, and mostly for taking interest to make open source code better. 
I merged your PR and released new patch to NPM. I have manually tested it, but it would be better if you would test it too and then close this issue.
BTW good job, I didn't even noticed this bug which reminds me this project misses automatic testing, I may implement it eventually in future, but I'm currently busy.

Best regards,
Filip.
